OS X 10.11.6 libjpeg fails to build

`Making all in java
make[3]: Nothing to be done for 'all'.
Making all in simd
/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-am
  CC       jsimd_arm.lo
In file included from ../jinclude.h:41:0,
                 from jsimd_arm.c:17:
/opt/devkitpro/devkitARM/arm-none-eabi/include/stdlib.h:253:27: error: macro "random" passed 1 arguments, but takes just 0
 long _EXFUN(random,(_VOID));
                           ^
`

If I remove the _VOID from that function then the error becomes

`/Applications/Xcode.app/Contents/Developer/usr/bin/make  all-am
  CC       jsimd_arm.lo
<command-line>:0:10: error: conflicting types for 'rand'
In file included from /opt/devkitpro/devkitARM/arm-none-eabi/include/stdlib.h:11:0,
                 from ../jinclude.h:41,
                 from jsimd_arm.c:17:
/opt/devkitpro/devkitARM/arm-none-eabi/include/stdlib.h:140:5: note: previous declaration of 'rand' was here
 int _EXFUN(rand,(_VOID));
     ^
`

In which I am not sure how to proceed.

--- a/Makefile
+++ b/Makefile
@@ -235,7 +235,7 @@ $(LIBJPEGTURBO): $(LIBJPEGTURBO_SRC)
 	@[ -d $(LIBJPEGTURBO_VERSION) ] || tar -xzf $<
 	@cd $(LIBJPEGTURBO_VERSION) && \
 	 ./configure --prefix=$(PORTLIBS_PATH)/armv6k --host=arm-none-eabi --disable-shared --enable-static
-	@$(MAKE) CFLAGS+="\"-Drandom()=rand()\"" -C $(LIBJPEGTURBO_VERSION)
+	@$(MAKE) -C $(LIBJPEGTURBO_VERSION)
 
 $(LIBMAD): $(LIBMAD_SRC)
 	@[ -d $(LIBMAD_VERSION) ] || tar -xzf $<

